influx/query.go

```diff
@@ -74,6 +75,11 @@ func Convert(influxQL string) (chronograf.QueryConfig, error) {
 		itsDashboardTime = true
 	}
 
+	if strings.Contains(influxQL, ":upperDashboardTime:") {
+		influxQL = strings.Replace(influxQL, ":upperDashboardTime:", "now() - 1m", 1)
+		itsUpperDashboardTime = true
```

Comment:
it feels like we are replacing strings back and forth- but when something looks this odd there's usually a good reason?

Reply:
the reason why we do this is so that the query will parse. It's a total hack.